mne_bids/tests/test_write.py

```diff
@@ -664,7 +664,7 @@ def test_vhdr(_bids_validate):
 
 
 @pytest.mark.filterwarnings(warning_str['nasion_not_found'])
-def test_edf(_bids_validate):
+def test_edf(_bids_validate, caplog):
```

Comment:
what's caplog?

Reply:
a pytest fixture for capturing log output, but this can be removed now. Sorry forgot to take this out.